changelog/split-std-datetime.dd

```diff
@@ -0,0 +1,43 @@
+std.datetime has been split into a package.
+
+std.datetime is now a package containing the following modules:
+
+$(UL
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_common.html">std.datetime.common</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_date.html">std.datetime.date</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_datetime.html">std.datetime.datetime</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_interval.html">std.datetime.interval</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_systime.html">std.datetime.systime</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_timeofday.html">std.datetime.timeofday</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_timezone.html">std.datetime.timezone</a>)
+)
+
+$(LI <a href="$(PHOBOS_PATH)std_datetime.html">std.datetime.package</a>)
```

Comment:
> All of the ddoc ref macros seems to use anchors and link to symbols in a module, not to the modules themselves, so I couldn't figure out which ddoc macros to use and just went with this. If anyone can show me how to do this with the ddoc ref macros, then I'll gladly change it.

The macros are `MREF` and `MREF_ALTTEXT`.

Examples:

* `$(MREF std,datetime,timezone)` makes a simple link using the FQN as the label.
* $(MREF_ALTTEXT std.datetime.package, std,datetime)` links to std.datetime using "std.datetime.package" as the label.